server/src/utils/fight/generateFight.ts

```diff
@@ -383,7 +397,8 @@ const generateFight = async ({
       clan.bossDamages.forEach((damage) => {
         userIds.add(damage.brute.userId || '');
       });
-      const goldGains = Math.floor(BOSS_GOLD_REWARD / userIds.size);
+      const goldReward = bossFighters.find((boss) => boss.name === BossName.Cerberus) ? FIRST_BOSS_GOLD_REWARD : BOSS_GOLD_REWARD;
+      const goldGains = Math.floor(goldReward / userIds.size);
```

Comment:
Instead of handling rewards case per case here, add a `reward` value to the defined boss list, set it to `1` for every other boss, and `0.X` for cerberus, this will be the reward ratio for each boss, to be used here to calculate the reward given, from a single `BOSS_GOLD_REWARD` ratioed (and rounded) result.